app/controllers/posts_controller.rb

```diff
@@ -4,7 +4,9 @@ def index
   end
 
   def show
-    @post = Post.includes(:comments, :likes).find(params[:id])
+    # @post = Post.includes(:comments, :likes).find(params[:id])
+    @post = Post.find(params[:id])
```

Comment:
- [ ] You need to use cancancan for post-authorization, so at the top of your post_controller file add this `load_and_authorize_resource`